tests/qos/test_pfc_pause.py

```diff
@@ -27,7 +27,7 @@
 
 PFC_PKT_COUNT = 1000000000
 
-PTF_FILE_REMOTE_PATH = '~/ptftests/pfc_pause_test.py'
+PTF_FILE_REMOTE_PATH = '~/ptftests/py3/pfc_pause_test.py'
```

Comment:
But the `pfc_pause_test.py` file is not moved to the `py3` sub-folder. Is it working?

Reply:
Thanks. Fixed it. I had missed the commit.